MetaMask/metamask-extension
# revert: reset prerender workaround

## **Description**

Reverts https://github.com/MetaMask/metamask-extension/pull/19727. The prerender regression in chromium has been resolved already. This fix is no longer required and is actually the cause of breaking streams for prerendered pages.

My bisection of chromium for the fix landed on this range of commits: https://chromium.googlesource.com/chromium/src/+log/55b4344edfb41dda980d197743f25a2841d498a4..c13107c16780c195bd5ec003d9198d87cdcc59dd

@shanejonas and I have verified that the prerender issue has been resolved in the latest chrome stable. I have verified that the workaround is causing an issue in develop that is no longer seen in this branch that reverts it.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24293?quickstart=1)

## **Related issues**

See: https://github.com/MetaMask/metamask-extension/pull/19727

## **Manual testing steps**
Using chrome, of course.
Visit `chrome://settings/?search=preload` and ensure it is enabled and on "standard"

1. On develop, make a build and load it
2. Visit `https://voyager-snap.linea.build/`
3. Install the snap and connect
4. Copy `https://voyager-snap.linea.build/` to your clipboard
5. Close all instances of `https://voyager-snap.linea.build/` that may be open
6. Rapidly, open a new tab, paste the url, hit enter, see if the page is stuck on "Loading", if not close the tab and repeat
7. Eventually the page should be get prerendered and stuck in a broken state

Do the same steps above, but using the changes in this branch. The page should never be stuck with "Loading"

## **Screenshots/Recordings**



### **Before**

https://github.com/MetaMask/metamask-extension/assets/918701/6bda7a5d-12d2-46e4-b5d6-8562c3143977


### **After**


https://github.com/MetaMask/metamask-extension/assets/918701/14289f0e-89eb-4534-8815-615a57e0386b



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
